components/alibi-detect-server/pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 [tool.poetry.dependencies]
 python = "^3.7"
 seldon-core = { path = "_seldon_core", develop = false }
-alibi-detect = "0.6.2"
+alibi-detect = "0.8.1"
 numpy = "1.19.5"
 cloudevents = "1.2.0"
 tensorflow = "2.4.2"
```

Comment:
This is slightly out of date, and `alibi-detect` is compatible with tf version up to 2.7, worth upgrading?

Can be left as a separate PR though.

Reply:
Yeah, this makes sense. I can try and see where it lead me - if no issues could go in here